cli/src/commands/services.js

```diff
@@ -47,20 +47,32 @@ async function postUpLaunch(spinner) {
     console.log('\n');
 }
 
+async function removeDynamicallyBuiltImages(){
+    const folderName = path.basename(fixDir())
+    const docker = new Docker({});
+    try {
+        await docker.command(`rmi ${folderName}_rasa ${folderName}_actions`);
+    } catch (e) {
+        // Do nothing - just in case the images do not exists
+    }
+}
+
 export async function doMinorUpdate() {
     const botfrontVersion = getBotfrontVersion();
     const projectVersion = getProjectVersion();
     const mongoPassword = getMongoPassword();
-    console.log(mongoPassword)
     if (isMajorUpdateWithVersion(projectVersion, botfrontVersion)){
         return console.log(boxen(`Project was made with Botfront ${chalk.blueBright(projectVersion)} and the currently installed version is ${chalk.green(botfrontVersion)}, which is a major update.\nPlease follow the instructions in the migration guide: ${chalk.cyan.bold('https://botfront.io/docs/migration')}.`));
```

Comment:
So I guess migration guide explains the need to and how to flush old images?

Reply:
good point...